rust/parquet/src/arrow/array_reader.rs

```diff
@@ -745,16 +1152,66 @@ impl<'a> TypeVisitor<Option<Box<dyn ArrayReader>>, &'a ArrayReaderBuilderContext
     }
 
     /// Build array reader for list type.
-    /// Currently this is not supported.
     fn visit_list_with_item(
```

Comment:
Is this being called anywhere?

Reply:
Yeah, it's called here: https://github.com/apache/arrow/blob/master/rust/parquet/src/schema/visitor.rs#L51